crates/uv/src/commands/build_backend.rs

```diff
@@ -31,19 +31,29 @@ pub(crate) fn build_wheel(
 }
 
 pub(crate) fn build_editable(
-    _wheel_directory: &Path,
-    _metadata_directory: Option<&Path>,
+    wheel_directory: &Path,
+    metadata_directory: Option<&Path>,
 ) -> Result<ExitStatus> {
-    todo!()
+    let filename = uv_build_backend::build_editable(
+        &env::current_dir()?,
+        wheel_directory,
+        metadata_directory,
+        uv_version::version(),
+    )?;
+    println!("{filename}");
```

Comment:
Is it definitely intended for this function to just write to stdout? If so, I might put that in the docs for the function. It's hard to articulate why, but it does seem somewhat surprising. I think part of it is that I don't quite understand why `filename` is being written to stdout. Is that just part of PEP 660? (I'm not familiar with it.)

Reply:
It's the way in PEP 517 and PEP 660 how the build backend tells the frontend where to find the build artifact: It's in `<wheel_directory>/<last line of stdout>`.

I'm adding references to the spec name in the doc comment, otherwise I'm assuming familiarity with the spec (sorry - i don't expect you to read them and will of course answer questions for you). I tried to copy all relevant excerpts from the spec in earlier PEP implementations I did, but didn't found it to be effective.